### ISSUE TITLE
Call to undefined function Prettus\Repository\Generators\app_path()

### ISSUE THREAD

**Issue by deneuv34** [andersao/l5-repository#472]
Hi, I'm installed this library on Lumen 5.5. I have register the service provider: `$app->register(Prettus\Repository\Providers\LumenRepositoryServiceProvider::class);`  to `bootstrap/app.php`.
But, when I try to create new repository I got error 
```php
[Symfony\Component\Debug\Exception\FatalThrowableError]              
  Call to undefined function Prettus\Repository\Generators\app_path() 
```
how to fix this, I try to change the `app_path()` to other helper like `app()->path()` but the error is nothing changes.

Thank you

**Comment by Maras0830** [comment]
@deneuv34 

because lumen did not have app_path() this helper.

https://github.com/andersao/l5-repository/pull/473

**Comment by deneuv34** [comment]
@Maras0830 as of your reference PR. The helper for lumen is `app()->path`? not `app()->path()` ?

**Comment by Maras0830** [comment]
@deneuv34 Thanks, that's good idea, although app()->path can get same result
and I found these code, we can call app()->basePath() to get more path on the project.

in vendor/laravel/framework/src/Illuminate/foundation/Application.php
```
        $this->instance('path', $this->path());
        $this->instance('path.base', $this->basePath());
        $this->instance('path.lang', $this->langPath());
        $this->instance('path.config', $this->configPath());
        $this->instance('path.public', $this->publicPath());
        $this->instance('path.storage', $this->storagePath());
        $this->instance('path.database', $this->databasePath());
        $this->instance('path.resources', $this->resourcePath());
        $this->instance('path.bootstrap', $this->bootstrapPath());
```

**Comment by eullercdr** [comment]
Fixed in 2.6.28. @Maras0830 Thanks so much one more time!
